packages/form/src/widgets/TextareaWidget/TextareaWidget.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+import { TextField, Icon } from '@equinor/eds-core-react'
+
+import { error_filled } from '@equinor/eds-icons'
+
+Icon.add({ error_filled })
+
+const TextareaWidget = (props: any) => {
```

Comment:
```suggestion
const TextareaWidget = (props:{ label: string, onChange: (value: any) => void }) => {
```

Reply:
Nice, I added a type for all widgets:

<img width="606" alt="image" src="https://user-images.githubusercontent.com/1190419/216120330-1f1b24c6-3b2f-4070-a505-b3f25354acdf.png">